include/fiction/algorithms/simulation_sidb/check_groundstate.hpp

```diff
@@ -0,0 +1,38 @@
+//
+// Created by Jan Drewniok on 18.01.23.
+//
+
+#ifndef FICTION_CHECK_GROUNDSTATE_HPP
+#define FICTION_CHECK_GROUNDSTATE_HPP
+
+#include "fiction/algorithms/simulation_sidb/ExGS.hpp"
+#include "fiction/algorithms/simulation_sidb/quicksim.hpp"
+#include "fiction/technology/charge_distribution_surface.hpp"
+
+namespace fiction
+{
+
+/**
+* @brief This function checks if the ground state function is found by the quicksim algorithm.
```

Comment:
@brief should not in the docstring. I tried to get rid of them all, but here I missed it.